framework/src/kernels/ConservativeAdvection.C

```diff
@@ -114,7 +133,9 @@ ConservativeAdvectionTempl<false>::computeQpJacobian()
 {
   // This is the no-upwinded version
   // It gets called via GenericKernel<false>::computeJacobian()
-  return negSpeedQp() * _phi[_j][_qp];
+  if (!_adv_quant_valid)
```

Comment:
Yea I think this data member could be renamed to something like `_user_supplied_adv_quant`